nsxt/resource_nsxt_transport_node.go

```diff
@@ -157,6 +159,7 @@ func getNodeSchema(addlAttributes map[string]*schema.Schema, addExactlyOneOf boo
 		"ip_addresses": {
 			Type:        schema.TypeList,
 			Optional:    true,
+			Computed:    true,
```

Comment:
I'm not sure about this. Is this field used by users to locate the node?

Reply:
Ah, I see what you mean. When read it back this field might contain more IP addresses.